expenseTracker/routes/expenses.js

```diff
@@ -0,0 +1,29 @@
+//defines routes for handling requests related to expenses
+const express = require("express");
+const router = express.Router();
+
+const {
+  getAllExpenses,
+  createExpense,
+  getSingleExpense,
+  updateExpense,
+  deleteExpense,
+} = require("../controllers/expenseController");
+const verifyToken = require("../middleware/auth");
+
+// Get All Expenses (GET /api/expenses)
+router.get("/", verifyToken, getAllExpenses);
+
+// Create an Expense (POST /api/expenses)
+router.post("/", verifyToken, createExpense);
+
+// Get a Single Expense (GET /api/expenses/:id)
+router.get("/:id", verifyToken, getSingleExpense);
+
+// Update an Expense (PATCH /api/expenses/:id)
+router.patch("/:id", verifyToken, updateExpense);
+
+// Delete an Expense (DELETE /api/expenses/:id)
+router.delete("/:id", verifyToken, deleteExpense);
```

Comment:
There is an argument to be made for doing it this way ahead of time in case you want any unprotected endpoints here, though.